### PR TITLE
Junit Changes

### PR DESCRIPTION
**Pull request by aniitavenkatesan**

### CODE REVIEW

File: src/test/java/org/recap/service/executor/datadump/AbstractDataDumpExecutorServiceUT.java [hunk 1]
```diff
@@ -108,7 +124,20 @@ public void testProcess() throws NoSuchMethodException, InvocationTargetExceptio
         }
 
     }
-    private String getDateTimeString() {
+
+    @Test
```
Comment (reachrama) [inline src/test/java/org/recap/service/executor/datadump/AbstractDataDumpExecutorServiceUT.java:128]: Remove System.out.print
Why are we having getFormaatedDateString and getDateTimeString() methods in the service level? Can this be move to some util classes?

File: src/test/java/org/recap/service/executor/datadump/DeletedDataDumpExecutorServiceUT.java [hunk 2]
```diff
@@ -210,11 +107,12 @@ public void testpopulateSearchRequest() {
...
-        mockedDeletedDataDumpExecutorService.populateSearchRequest(searchRecordsRequest, dataDumpRequest);
-        } catch (Exception e) {
-        e.printStackTrace();
-        }
-        assertTrue(true);
+        return searchRecordsRequest;
+    }
+
+    private String getDateTimeString() {
+        Date date = new Date();
```
Comment (reachrama) [inline src/test/java/org/recap/service/executor/datadump/DeletedDataDumpExecutorServiceUT.java:114]: I see the same method in the previous class - duplicate code try to minimize that

File: src/test/java/org/recap/service/executor/datadump/IncrementalDataDumpExecutorServiceUT.java [hunk 3]
```diff
@@ -1,97 +1,75 @@
...
+        searchRecordsRequest.setSelectAllFacets(true);
+        searchRecordsRequest.setShowTotalCount(true);
+        searchRecordsRequest.setIndex(1);
+        searchRecordsRequest.setDeleted(false);
+        searchRecordsRequest.setErrorMessage("test");
+        return searchRecordsRequest;
+    }
+
+    private String getDateTimeString() {
+        Date date = new Date();
```
Comment (reachrama) [inline src/test/java/org/recap/service/executor/datadump/IncrementalDataDumpExecutorServiceUT.java:68]: Same here, duplicate method

**Review by reachrama — COMMENTED** [review]
Please review the comments

**Review by reachrama — APPROVED** [review]
Looks good

**Comment by aniitavenkatesan** [comment]
Closing this PR. Will create new PR with suggested changes.

**Pull request closed without merging**